challenges/functions.js

```diff
@@ -6,25 +6,29 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+const consume = (frick, frack, cb) => {return cb(frick, frack)}
 
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+const add = (frick, frack) => {return cb(frick, frack)}
+const multiply = (frick, frack) => {return cb(frick, frack)}
+const greeting = (frick, frack) => {return 'Hello ${frick} ${frack}, nice to meet you!' }
```

Comment:
If You want to take advantage of ES6 String Literals You need `backtick` - not a quote. Hence the code should look like this:
```
const greeting = (frick, frack) => {return `Hello ${frick} ${frack}, nice to meet you!` }
```